osg-ca-certs-updater.py

```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/python
 """OSG Auto-Updater for CA Certificates"""
 # (ignore bad name of script) pylint: disable=C0103
+from __future__ import absolute_import
```

Comment:
Just curious, which module(s) use `absolute_import` ?

Reply:
Ugh. None of them.

Reply:
Wow, yeah, it looks so modern now.  Absolutely from the future.